### PR TITLE
[CSL-884] Investigate alternative to pre-push

### PR DESCRIPTION
**Pull request by esezen**
To test:
* Create a lint issue in code
* Try pushing to this branch
* Push should fail

### CODE REVIEW

**Review by crgee1 — APPROVED** [review]
lgtm

File: .husky/pre-push [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```
Comment (jjl014) [inline .husky/pre-push:2]: Just for my curiousity. What does this do?

**Review by jjl014 — APPROVED** [review]
Nice, lgtm!

File: .husky/pre-push (hunk 1 above)
Comment (esezen) [reply .husky/pre-push:2]: Not sure tbh. It is automatically created by husky per hook (push, commit etc).

https://github.com/typicode/husky
```
npx husky add .husky/pre-commit "npm test"
git add .husky/pre-commit
```

Comment (jjl014) [reply .husky/pre-push:2]: Oooo, interesting. And I'm guessing you just add any npm commands you want to be ran pre-push into this file?

Comment (esezen) [reply .husky/pre-push:2]: Yessir

**Comment by sblaurock** [comment]
Works great!

**Comment by sblaurock** [comment]
Can we port to constructorio-node as well?

**Comment by esezen** [comment]
> Can we port to constructorio-node as well?

Hmm I thought I did but it looks like I missed that one. Will do

**Pull request merged**